src/wallet/rpcwallet.cpp

```diff
@@ -3166,6 +3166,13 @@ Value z_getoperationstatus_IMPL(const Array& params, bool fRemoveFinishedOperati
     return ret;
 }
 
+// Here we define the maximum number of zaddr outputs that can be included in a transaction.
+// If input notes are small, we might actually require more than one joinsplit per zaddr output.
+// For now though, we assume we use one joinsplit per zaddr output (and the second output note is change).
+// We reduce the result by 1 to ensure there is room for non-joinsplit CTransaction data.
+#define Z_SENDMANY_MAX_ZADDR_OUTPUTS    ((MAX_TX_SIZE / 1802) - 1)  // Serialized JSDescription is 1802 bytes
```

Comment:
I would rather use `GetSerializeSize()` on a blank `JSDescription` (they are constant size) than forget to modify this constant. :)